Flabbergasted-Flamingos/Bangazon-admin-app
# Rb navigation

# Description
This is the nav_bar.html. It is in all the currently written templates as an include statement. 
I had to write an index view and also added an index.html


## Related Ticket(s)
all

## Problem to Solve
This makes bangazon more navigatable and beautiful

## Proposed Changes
Affects views, init.py for views, templates and static files

## Expected Behavior
When you click on employees and departments it will bring up those lists

## Steps to Test Solution

1. 
```
git fetch --all
git checkout rb_navigation
python manage.py runserver
```
2. Have a look at it, play with the buttons and make sure it's working.
